include/sacn/cpp/source.h

```diff
@@ -121,6 +121,10 @@ class Source
     /** If true, this sACN source will only send unicast traffic on this universe. Defaults to false. */
     bool send_unicast_only{false};
 
+    /** The initial set of unicast destinations for this universe. This can be changed further by using
+        Source::AddUnicastDestination() and Source::RemoveUnicastDestination(). */
+    const std::vector<EtcPalIpAddr> unicast_destinations;
```

Comment:
Had to do a little bit of legwork since I'm storing a private vector of `EtcPalIpAddr` since that memory needs to live somewhere. See if you like what I wrote.

Reply:
@ChristianReese  Take a look at how it's done in RDMnet: https://github.com/ETCLabs/RDMnet/blob/develop/include/rdmnet/cpp/device.h#L479

Reply:
Good to know for the future. Nick made me realize that this memory will only be used temporarily, but I've bookmarked this for future reference.

Reply:
Backtracking again. The memory still needs a place to live so I'm going with the RDMnet implementation.